pkg/bzz/address.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package bzz
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/binary"
+	"encoding/json"
+	"errors"
+
+	"github.com/ethersphere/bee/pkg/crypto"
+	"github.com/ethersphere/bee/pkg/swarm"
+
+	ma "github.com/multiformats/go-multiaddr"
+)
+
+var ErrInvalidAddress = errors.New("invalid address")
+
+type Address struct {
```

Comment:
it would be nice to have a godoc on this type, with explanation on how what the signature represents